zipkin-server/README.md

```diff
@@ -115,6 +115,7 @@ zipkin-server is a drop-in replacement for the [scala query service](https://git
     * `QUERY_LOOKBACK`: How many milliseconds queries can look back from endTs; Defaults to 24 hours (two daily buckets: one for today and one for yesterday)
     * `STORAGE_TYPE`: SpanStore implementation: one of `mem`, `mysql`, `cassandra`, `elasticsearch`
     * `COLLECTOR_SAMPLE_RATE`: Percentage of traces to retain, defaults to always sample (1.0).
+    * `HTTP_COLLECTOR_ENABLED`: `false` disables the http collector; Defaults to true
```

Comment:
adding QUERY_ENABLED to this section was probably a bit of a mistake as that env never existed before. How about adding a new section based on `Scribe Collector`? In the new section, I'd explain this accepts spans via `POST /api/v1/spans`

I feel awkward about `HTTP_COLLECTOR_ENABLED` as much as `HTTP_ENABLED` almost to the point of punting re-mapping of the spring property. That said, it isn't a strong feeling.